app/react/Map/Map.js

```diff
@@ -321,6 +323,13 @@ export default class Map extends Component {
     return false;
   }
 
+  async replaceKeysMapStyleJson() {
+    const mapTemplateEntry = /{{MAP_TILER_KEY}}/g;
```

Comment:
I don't really understand this variable name?  should this be `mapTilerKeyPlaceholder`, or something in those lines?

Reply:
mapTilerKeyPlaceholder is perfect 👍